templates/export.html.twig

```diff
@@ -1,11 +1,15 @@
-{% extends '@EasyAdmin/default/layout.html.twig' %}
-
+{% extends '@!EasyAdmin/layout.html.twig' %}
 {% block page_title %}
   {{ 'export.headline'|trans }}
 {% endblock %}
 
 {% block main %}
+
+
+
   <div class="container-fluid">
+
+
     <h1>{{ 'export.headline' | trans }}</h1>
```

Comment:
```suggestion
{% block main %}
  <div class="container-fluid">
    <h1>{{ 'export.headline' | trans }}</h1>
```

Reply:
@Mr-Martin-Kristiansen, consider adding https://github.com/VincentLanglet/Twig-CS-Fixer as a development dependency to clean up the Twig templates. We use that tool in several projects, e.g. https://github.com/itk-dev/aapodwalk_api/ (see also https://github.com/search?q=org%3Aitk-dev%20vincentlanglet%2Ftwig-cs-fixer&type=code).